firmware/src/rdm_responder.c

```diff
@@ -392,6 +398,9 @@ int RDMResponder_GetSupportedParameters(const RDMHeader *header,
       case PID_SOFTWARE_VERSION_LABEL:
       case PID_DMX_START_ADDRESS:
       case PID_IDENTIFY_DEVICE:
+        if (g_responder->is_subdevice) {
```

Comment:
Section 2 of E1.37-1 "Sub-devices should always declare all their supported parameters in their list of Supported Parameters. "

I've opened a bug to add a RDM Responder tests for this issue.